hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -163,27 +166,12 @@
       stage: V2
       tests: ["chip_sw_spi_device_pass_through"]
     }
-    {
-      name: chip_sw_spi_device_pass_through_filter
-      desc: '''Verify the command filtering mechanism in passthrough mode.
-
-            - Extend the chip_spi_device_pass_through test.
-            - Program the cmd_filter_* CSRs to filter out certain commands. a one byte command (such
-              as Chip Erase) and a multi-bytes command (such as Program or Sector/Block Erase) must
-              be added to filter
-            - Verify that only the payloads that are not filtered show up on the SPI host interface
-              at chip IOs.
-            '''
-      stage: V2
-      tests: []
-    }
 
     {
       name: chip_sw_spi_device_pass_through_flash_model
       desc: '''Verify the command filtering mechanism in passthrough mode.
 
-            - Extend the chip_spi_device_pass_through and chip_spi_device_pass_through_filter
-              tests.
+            - Extend the chip_spi_device_pass_through test.
             - Connect with a real flash model on spi_host
```

Comment:
we don't have a real flash model yet. We may need deprioritize this test.

Perhaps we can develop `chip_sw_spi_device_pass_through_collision` first.